gdk4-win32/Cargo.toml

```diff
@@ -27,13 +27,13 @@ egl = ["khronos-egl"]
 win32 = ["windows"]
 
 [dependencies]
-ffi = {path = "./sys", package = "gdk4-win32-sys", version = "0.5.2"}
-gdk = {path = "../gdk4", package = "gdk4", version = "0.5.2"}
+ffi = {path = "./sys", package = "gdk4-win32-sys", version = "0.5.3"}
+gdk = {path = "../gdk4", package = "gdk4", version = "0.5.3"}
 gio = {git = "https://github.com/gtk-rs/gtk-rs-core", branch = "0.16", version = "0.16", features = ["v2_66"]}
 glib = {git = "https://github.com/gtk-rs/gtk-rs-core", branch = "0.16", version = "0.16", features = ["v2_66"]}
 libc = "0.2"
 khronos-egl = {version = "4.1.0", optional = true}
-windows = { version = "0.41.0", features = [
+windows = { version = "0.43.0", features = [
```

Comment:
Does this crate export types from the windows crate? If so this would be a breaking change.

Reply:
it does, only if you opt-in for the windows feature, otherwise we provide our own types. I think @elmarco is the only one using this crate anyway, would it be fine to you to backport this change?

Reply:
Seems ok to me if @elmarco is happy with it

Reply:
Alright, let me drop it :)

Reply:
Hmmm, I am sure i didn't backport that commit

Reply:
see https://github.com/gtk-rs/gtk4-rs/blob/0.5/gdk4-win32/Cargo.toml#L36